src/frontends/onnx/frontend/src/input_model.cpp

```diff
@@ -193,6 +193,27 @@ void InputModel::set_element_type(const ov::frontend::Place::Ptr& place, const n
     m_editor->set_input_types(m);
 }
 
+ov::element::Type InputModel::get_element_type(const ov::frontend::Place::Ptr& place) const {
+    std::string tensor_name;  // name of the model input which should be reshaped
```

Comment:
nit: The comment about reshape seems not related to the `tensor_name` usage in `get_element_type`
```suggestion
    std::string tensor_name;
```